.eslintrc.cjs

```diff
@@ -130,6 +130,15 @@ module.exports = {
         'prefer-declare-function': 'error',
         'redundant-undefined': 'error',
         'trim-file': 'error',
+        '@typescript-eslint/ban-ts-comment': [
+            'error',
+            {
+                'ts-expect-error': false, // Used in tests.
+                'ts-ignore': 'allow-with-description',
```

Comment:
I could leave this as `true` and regular eslint ignores could be used, though it's a bit nice because this gives a nice error that asks the user to explain what they're doing:

https://typescript-eslint.io/play/#ts=4.8.4&fileType=.tsx&code=LAKA9GAEACAuDOBaAlgcwHYHsBOBTUsAngA66QCiANrgLa7qwAqJZAvJAIbqGS4AesegBN4kAFIBlABoA6KrXpMWkAPyR0uAG65skAFzjpc6nQbNSAbiA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQAIBcBPABxQGNoBLY-AWhXkoDt8B6AIwEMnb9layAewC2wxC3RgA2uGxy5kRNGiDokADSy5WeboV96AD1Jk6SlWowAzTvGSJNep1AOUA5k1WJJkW-EEA7rQBlPgAFrQAJuRUNJSCTBpazvr8nmRhiGQA1pL4cA7JKTgGGVm51rb2RWAAvskAuo51ILVAA&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA&tokens=false